pkg/aws/bucket_cache.go

```diff
@@ -56,11 +59,18 @@ func newBucketCache(s *storage) (*bucketCache, error) {
 	// us-east-1 is being used here because it is the default region but plays no role in
 	// listing the buckets, it is dependent on the account that created the s.s3 object.
 	region := "us-east-1"
+	log.Debug("BP1")
 	v := s.s3[region]
+	for key, val := range s.s3 {
```

Comment:
Lines 62, 64-67, 69 and 73 are leftover from debugging, they can be removed.

Reply:
I will clean these up in my next code review.